packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md

```diff
@@ -2053,7 +2053,7 @@ HTTP 204 No Content
 ### Update a Server on a Project
 
 <ApiOperation method="PUT" url="https://app.scaleft.com/v1/teams/${team_name}/projects/${project_name}/servers/${server_id}" />
-Updates a Server on a Project. This endpoint is only available with the PolicySync feature, which is currently in EA.
+Updates a Server on a Project. This endpoint is intended for use with the PolicySync feature, see [PolicySync: Attribute-Based Access Control](https://help.okta.com/okta_help.htm?type=asa&id=ext-asa-policysync).
```

Comment:
```suggestion
Updates a Server on a Project. This endpoint is used for the PolicySync feature, see [PolicySync: Attribute-Based Access Control](https://help.okta.com/okta_help.htm?type=asa&id=ext-asa-policysync).
```
The help doc alias is not available yet?

Reply:
Should be live this week.